onnx/defs/tensor/defs.cc

```diff
@@ -2901,20 +2901,19 @@ ONNX_OPERATOR_SET_SCHEMA(
         }));
 
 static const char* Where_ver16_doc = R"DOC(
-    Return elements, either from X or Y, depending on condition
-    (with Numpy-style broadcasting support).
-    Where behaves like numpy.where with three parameters:
-    https://docs.scipy.org/doc/numpy/reference/generated/numpy.where.html
+    Return elements, either from X or Y, depending on condition.
+    Where behaves like
+    [numpy.where](https://docs.scipy.org/doc/numpy/reference/generated/numpy.where.html)
+    with three parameters.
```

Comment:
Thank you for proposing this PR. Basically looks good to me. Shall we apply this doc rewording to Where-9 as well?